src/index.js

```diff
@@ -0,0 +1,112 @@
+let count = 80;
+
+// const state = {
+//     count = 80,
+//     landscape = "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂"
+// }
+
+const increaseTemp = document.getElementById("increaseTemp");
+const decreaseTemp = document.getElementById("decreaseTemp");
+const counter = document.getElementById("count");
+counter.innerHTML = count;
+
+// const temp = document.getElementById('temp');
+// function increaseTemp() {
+//     temp.value = currentTemp
+//     currentTemp++;
+// }
+
+// function decreaseTemp() {
+//     temp.value = currentTemp
+//     currentTemp--;
+// }
+
+// function changeColor(newColor) {
+//     const elem = document.getElementById("count");
+//     elem.style.color = newColor
+//     // elem.style.backgroundColor = newColor
+// }
+
+// }
+// function updateColorAndLandscapeByTemp(updatedTemp) {
+//     switch(updatedTemp){
+//         case (updatedTemp >= 80):
+//             // count.color = 'red';
+//             changeColor('red');
+//             landscape.emoji = "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂";
+//             break;
+//         case (updatedTemp >= 79):
+//             // count.style.color = 'orange';
+//             changeColor('orange');
+//             landscape.emoji = "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷";
+//             break;
+//         case (updatedTemp >= 69):
+//             // count.style.color = 'yellow';
+//             changeColor('yellow');
+//             landscape.emoji = "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃";
+//             break;
+//         case (updatedTemp >= 59):
+//             // count.style.color = 'green';
+//             changeColor('green');
+//             landscape.emoji = "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲";
+//             break;
+//         case (updatedTemp >= 49):
+//             // count.color = 'teal';
+//             changeColor('teal');
+//             break;
+//     }
+// }
+
+// const sky = document.getElementById("sky");
+// let sky_text = "☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️";
+// const sky = document.querySelector('#sky');
+// const updateSkyEmoji = (updatedSky) => {
+//     switch(updatedSky) {
+//         case (updatedSky == "Sunny") {
```

Comment:
We don't write a conditional for each `case` clause. Instead, we provide a value that will be compared to the value provided in the `switch` expression.
```js
         case "Sunny" {
```